scripts/utils/variables.py

```diff
@@ -17,15 +17,12 @@
 GAS_FEE_PERCENTILE_1: int = int(os.getenv('GAS_FEE_PERCENTILE_1', 5))
 GAS_FEE_PERCENTILE_DAYS_HISTORY_1: int = int(os.getenv('GAS_FEE_PERCENTILE_DAYS_HISTORY_1', 1))
 
-GAS_FEE_PERCENTILE_2: int = int(os.getenv('GAS_FEE_PERCENTILE_2', 20))
-GAS_FEE_PERCENTILE_DAYS_HISTORY_2: int = int(os.getenv('GAS_FEE_PERCENTILE_DAYS_HISTORY_2', 2))
-
 GAS_PRIORITY_FEE_PERCENTILE = int(os.getenv('GAS_PRIORITY_FEE_PERCENTILE', 55))
 
 MIN_PRIORITY_FEE = Wei(os.getenv('MIN_PRIORITY_FEE', '2 gwei'))
 MAX_PRIORITY_FEE = Wei(os.getenv('MAX_PRIORITY_FEE', '10 gwei'))
 
-MAX_BUFFERED_ETHERS = Wei(os.getenv('MAX_BUFFERED_ETHERS', '5000 ether'))
+MAX_BUFFERED_ETHERS = Wei(os.getenv('MAX_BUFFERED_ETHERS', '7000 ether'))
```

Comment:
Why 7k? We agreed on 5k and we have an alert for this made by the automation team